### DISCUSSION TITLE
無法開啟

### DISCUSSION THREAD

**Discussion by saunpcnaye** [MaaAssistantArknights/MaaAssistantArknights#3135] [Q&A]
![dll](https://user-images.githubusercontent.com/108626172/208710095-60fc7bf6-ddc2-4d2d-a16a-b465a5277f70.PNG)

**Comment by ABA2396** [comment]
看下issues里的置顶有没有帮助 #3083

**Reply by saunpcnaye** [reply]
win7遇到的問題

**Comment by ABA2396** [comment]
![Screenshot_2022-12-21-01-51-55-036_com tencent mobileqq](https://user-images.githubusercontent.com/99072975/208733446-d3aa2428-fff8-4b2c-8722-40cc69d92ee1.jpg)

**Reply by saunpcnaye** [reply]
感謝 剛剛滑下去沒看清
